Client/src/app/components/youTubeIframeComponent/youTubeIframe.component.ts

```diff
@@ -0,0 +1,44 @@
+import {Component, Input} from '@angular/core';
+import { Injectable } from '@angular/core';
+import {PlayerService} from '../player.service';
+
+@Component({
+  selector: 'iframe-component',
+  templateUrl: './youTubeIframe.component.html',
+})
+
+@Injectable()
+export class YouTubeIframeComponent {
+  isReady = false;
+  @Input() videoId;
+  public YT;
+
+  constructor (private playerService: PlayerService) {
+  }
+
+  init() {
+    const tag = document.createElement('script');
```

Comment:
is that's how you create the iframe? make rename init() to something like buildYoutubeApiIframe()?